.github/workflows/github-actions.yml

```
@@ -7,8 +7,8 @@ jobs:
     name: run test
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/cache@v2
+      - uses: actions/checkout@v4 # Updated
+      - uses: actions/cache@v3   # Updated
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)